### DISCUSSION TITLE
OpenFast simulation for airborne wind energy systems

### DISCUSSION THREAD

**Discussion by HongYue-Strathclyde** [OpenFAST/openfast#1617] [Q&A]
I'm modelling an airborne wind energy system using OpenFast. I'd like to get more information on the use of AWT_Yfixed and/or AWT_Yfree. To start with, I'd like to use one of them to congigurate my own prototype system, Any information would be appreciated. 

Many thanks,
Hong

**Comment by jjonkman** [comment]
Dear @HongYue-Strathclyde,

The AWT models are for the AWT-27CR2 2-bladed, 175-kW, 27-m rotor diameter horizontal axis wind turbines, which are not airborne.

Perhaps KiteFAST is of interest to you for fly-gen-type fixed-wing airborne wind energy systems: https://github.com/OpenFAST/KiteFAST?

Best regards,

**Reply by HongYue-Strathclyde** [reply]
Hi @jjonkman, thank you for the quick response, very helpful to clear out my wrong ideas. I'll check KiteFAST and see if it works for ground-gen-type soft-wing airborne wind energy systems.

Best regards,
Hong

**Reply by jjonkman** [reply]
Sorry, but KiteFAST was developed mainly for fly-gen-type fixed-wing airborne wind energy systems.

Best regards,
